backend/ceo_router.py

```
@@ -0,0 +1,142 @@
+                "max_attempts_per_subtask": config.max_attempts_per_subtask,
+                "escalation_enabled": config.escalation_enabled,
+                "llm_decomposition": config.llm_decomposition,
+                "require_tests_for_code": config.require_tests_for_code,
+            },
+        }
+
+    @router.get("/goals")
+    async def list_goals(
+        state: str | None = Query(default=None),
+        limit: int = Query(default=25, ge=1, le=200),
+        user: dict = Depends(get_current_user),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)